api/application.proto

```diff
@@ -83,6 +83,12 @@ message ListApplicationAPIKeysRequest {
   uint32 page = 3;
 }
 
+message GetApplicationAPIKeyRequest {
+  ApplicationIdentifiers application_ids = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false, (validate.rules).message.required = true];
+  // Unique public identifier for the API key.
+  string key_id = 2 [(gogoproto.customname) = "ID"];
```

Comment:
The customname should be consistent with `key_id`. Better to use `KeyID`. Same in other files.